### ISSUE TITLE
Removing unnecessary React imports

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#951]
We are now using the new jsx transform

- Use the script here to update files: https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports
- Add "plugin:react/jsx-runtime" to "extends" in eslint config: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md

This issue needs to be done after #918 as these changes would touch a lot of files.

**Comment by Barsnes** [comment]
We can also use this tool to find unused files, dependencies and imports.
https://github.com/webpro/knip

**Comment by mimarz** [comment]
Prioritising this as its needed to make sure bachelor assignment is coded using our configuration to enable smoother handover.

**Comment by mimarz** [comment]
> We can also use this tool to find unused files, dependencies and exports. https://github.com/webpro/knip

Made a separate issue for this, [#1502](https://github.com/digdir/designsystemet/issues/1502) 😋
